json_schemas/services-g-cloud-7-iaas.json

```diff
@@ -444,7 +444,10 @@
       "type": "object",
       "properties": {
         "value": {
-          "type": "number"
+          "type": "number",
+          "minimum": 0,
+          "maximum": 100,
+          "exclusiveMaximum": true
```

Comment:
100% uptime is a lie :) 
We didn't allow 100% for G6 - so no existing services would fail to validate against this (and these are G7 schemas anyway and won't be applied to existing services).